src/xenia/gpu/command_processor.h

```diff
@@ -171,7 +171,10 @@ class CommandProcessor {
   virtual void WriteRegistersFromMem(uint32_t start_index, uint32_t* base,
                                      uint32_t num_registers);
 
+  //FIXME(RodoMa92): This fails on Clang with Linux
+#ifdef XE_PLATFORM_WINDOWS
```

Comment:
Shouldn't it fail on compilation? I think CI for linux uses Clang and compilation goes smoothly (I'm ignoring linker issues for now)

Reply:
It does on my end:
xenia-canary/src/xenia/gpu/command_processor.h: In member function ‘bool xe::gpu::vulkan::VulkanCommandProcessor::ExecutePacketType3_SET_CONSTANT2(uint32_t, uint32_t)’:
/home/marco/Sviluppatore/xenia-canary/src/xenia/gpu/command_processor.h:176:16: error: inlining failed in call to ‘always_inline’ ‘virtual void xe::gpu::CommandProcessor::WriteRegisterRangeFromRing(xe::RingBuffer*, uint32_t, uint32_t)’: function body not available
  176 |   virtual void WriteRegisterRangeFromRing(xe::RingBuffer* ring, uint32_t base,